<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MedPayment</class>
 <widget class="QMainWindow" name="MedPayment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>822</width>
      <height>44</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {background-color:rgb(66, 57, 57); color:rgb(255, 255, 255);
}
</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Last name</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="pushButton_5">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {color: white; border-radius: 6px;min-width: 80px;}</string>
         </property>
         <property name="text">
          <string>Log out</string>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>First name</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="pushButton_4">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {color: white; border-radius: 6px;min-width: 80px;}</string>
         </property>
         <property name="text">
          <string>My Account</string>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="WalletDiscountLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>E_Wallet discount: </string>
    </property>
   </widget>
   <widget class="QLabel" name="discountAmount">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>220</y>
      <width>49</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>800</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>Type of Payment:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="typeOfPaymentToDisplay_2">
       <property name="text">
        <string>Medicine from pharmacy</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Payment</string>
    </property>
   </widget>
   <widget class="QLabel" name="outstandingPayLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>240</y>
      <width>49</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>800</string>
    </property>
   </widget>
   <widget class="QLabel" name="outstandingAfterDiscountAmount">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>260</y>
      <width>49</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>800</string>
    </property>
   </widget>
   <widget class="QLabel" name="outstandingAfterDiscountLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Outstanding pay after discount: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Outstanding pay: </string>
    </property>
   </widget>
   <widget class="QPushButton" name="PayButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>370</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Pay</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>170</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Payment method</string>
    </property>
   </widget>
   <widget class="QLabel" name="CashPromptLabel">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>210</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Please pay at the reception</string>
    </property>
   </widget>
   <widget class="QLabel" name="paymentSuccessfulLabel">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>400</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Payment successful.</string>
    </property>
   </widget>
   <widget class="QLabel" name="CVVError">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>280</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="messageToUser">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>400</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>random text</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>210</y>
      <width>203</width>
      <height>136</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <widget class="QLabel" name="CardNumberLabel">
       <property name="text">
        <string>Card number</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="CVVTxbx"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="CardNameLabel">
       <property name="text">
        <string>Card name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="CardNumberTxbx"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="CardNameTxbx"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="ExpiryDateLabel">
       <property name="text">
        <string>Expiry date</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="CVVlabel">
       <property name="text">
        <string>CVV</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDateEdit" name="ExpDateEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="redeemPointsMessage">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>110</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QComboBox" name="paymentMethodCombo">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>170</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>E-Wallet</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Insurance</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cash</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Credit</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="balanceToDisplay">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>210</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="cardNumberError">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>230</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="BalanceLabel">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>210</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Balance:</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>70</y>
      <width>186</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_13">
       <property name="font">
        <font>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>Points:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="points">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="redeemPointsButton">
       <property name="text">
        <string>Redeem points</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>320</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>*in this case, the outstanding pay is equivalent to
the total price of medicines you bought. </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>826</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
